build_tools/kokoro/gcp_ubuntu/docker_run.sh

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 The IREE Authors
```

Comment:
new file could use 2022

Reply:
It's forked from an old file and we're supposed to leave the oldest copyright year :shrug:

Reply:
Fair enough. It just seemed simple enough that it could be new and not forked :P

Reply:
It could've been, but it wasn't :-P